process/request_resources.md

```diff
@@ -0,0 +1,60 @@
+# Requesting Resources
+
+All OpenSSF projects receive support and resources from the Foundation. The type of resources and amount vary by project stage; refer to the [project lifecycles](process/project_lifecycle.md) for details. 
+
+- Common requests
+    - [Blogs](#communications-and-content)
+    - [Event space](#contribuor-meet-ups-at-OpenSSF-events)
+    - [Infrastructure](#infrastructure)
+    - [Security audits](#security-audits-and-improvements)
+    - [Code of Conduct support](#code-of-conduct-support)
+
+## Requests through the OpenSSF Marketing Committee
+
+### Communications and content
+
+To post on the OpenSSF blog, projects should contact the OpenSSF Marketing Committee with a blog draft two weeks before the requested publication date. The Committee does their best to accommodate requested publication dates, but projects should plan to be flexible during high-traffic times (e.g. events). Posts and timing is at the editorial discretion of the Marketing Committee; posts must follow the Code of Conduct, be relevant to the OpenSSF mission, and not promote vendor or commercial products. 
```

Comment:
The Marketing Committee is not critical-path to publishing on the blog. OpenSSF Staff schedule and shepherd posts to publication, with input from everyone and overall guidance from the MC and others welcome. I'm not actually sure whether we need this section in the "request resources.md" document, to be honest. Or at least, can we work these issues out in a separate PR?